src/components/switch/__internal__/switch-slider.style.js

```diff
@@ -7,7 +7,7 @@ import StyledValidationIcon from "../../../__internal__/validations/validation-i
 
```

Comment:
I guess you could also get rid of the `import baseTheme from "../../../style/themes/base";` since StyledSwitchSlider no longer needs default `theme` prop